src/web/waiters/InputWaiter.mjs

```diff
@@ -91,7 +91,7 @@ class InputWaiter {
             lineWrapping: new Compartment,
             fileDetailsPanel: new Compartment
         };
-
+        localStorage.setItem("alerted", "no");
```

Comment:
Maybe best to change this to `this.alerted = 'no'` as opposed to using local storage.